.github/workflows/sanitize.yaml

```diff
@@ -0,0 +1,80 @@
+name: Sanitize with AddressSanitizer, MemorySanitizer, ThreadSanitizer and UndefinedBehaviorSanitizer.
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build_dependencies:
+    name: Build deps [ubuntu]
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'APPROVED'
```

Comment:
Is the state case sensitive?

Reply:
In [github actions doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-when-a-pull-request-is-approved) it is in uppercase , but I saw also in lowercase, e.g. [here](https://stackoverflow.com/questions/69493260/get-approval-status-in-github-action)